src/components/cards/cards.comp.tsx

```diff
@@ -45,7 +48,12 @@ export const Cards = ({ items, fields, callbacks, customActions, customLabels }:
   const deleteLabel: string = customLabels?.buttons?.deleteItem || 'Delete';
 
   return (
-    <div className="cards-wrapper">
+    <InfiniteScroll className="cards-wrapper"
```

Comment:
It should wrap items only in case the user configured `lazy-loading`.

Reply:
Yeah, I got lazy to do so as it did not have any impact if not given its callbacks.

Reply:
That's fixed